app/assets/javascripts/application/codemirror-modes/alias-modes.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+
+  function CodeMirrorAlias(alias, current) {
+    CodeMirror.defineMIME(alias, CodeMirror.mimeModes[current]);
+  }
+
+  CodeMirrorAlias('c', 'text/x-csrc');
+  CodeMirrorAlias('cpp', 'text/x-c++src');
+  CodeMirrorAlias('c_cpp', 'text/x-c++src');
+
+  CodeMirrorAlias("html", "text/html");
+
+  CodeMirrorAlias('java', 'text/x-java');
+
+  CodeMirrorAlias('sql', 'text/x-sql');
+
+  CodeMirrorAlias('sh', 'text/x-sh');
+  CodeMirrorAlias('bash', 'text/x-sh');
```

Comment:
I don't find many missing modes here, should we add them?

* Prolog
* Haskell
* JavaScript
* Ruby
* Gobstones
* Wollok
* ASM/QSIM/Assembler
* Python
* Elixir

See http://runners.mumuki.io/runners